docker/docker-compose.ci.yml

```diff
@@ -6,11 +6,13 @@ services:
     expose:
       - 8545
     command: ["-h=0.0.0.0", "-m=hello unlock save the web"]
-  unlock:
-    build:
-      context: ..
-      dockerfile: ./docker/unlock.dockerfile
+  smart-contracts:
+    image: unlock
```

Comment:
This is more of a lack of understanding of docker on my part, but what occurs in this scenario of if the unlock image isn't currently available on this machine?

Reply:
So, this is a good question. I just tested and it docker-compose refuses to build. I guess this is expected.